doc/developer-guide/deployment/https.rst

```diff
@@ -3,9 +3,10 @@
 HTTPS support
 =============
 
-Zotonic has built-in support for https and certificate handling.
+Zotonic has built-in support for HTTPS and TLS (previously SSL) certificate
+handling.
 
-All sites share a single ssl port using Server Name Indication for 
+All sites share a single TLS port using Server Name Indication for
```

Comment:
I agree with using HTTPS where possible.

We can leave some *SSL* and *TLS* words in the text, so that people can search for them and find the correct pages.